### PR TITLE
Underscores

### PR DESCRIPTION
**Pull request by benjub**
Now that https://github.com/metamath/metamath-exe/pull/160 handling underscores has been merged, I made a pass on the quoted Metamath program commands in set.mm to uniformize them.  In the process, I saw some http URLs so I also uniformized them to `https://us.metamath.org/`.

### CODE REVIEW

File: iset.mm [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
+$( New users may want to read https://us.metamath.org/ileuni/conventions.html
+   to understand the label naming conventions used in iset.mm.  See also "help
    verify markup" in the Metamath program for markup conventions. $)
 
-$( To break this file into smaller modules, in the Metamath program type "read
-   iset.mm" followed by "write source iset.mm /split"; to recombine, omit
-   "/split". $)
+$( To break this file into smaller modules, in the Metamath program type
+   "MM> READ iset.mm" followed by "MM> WRITE SOURCE iset.mm / SPLI".  To
+   recombine, omit "/ SPLIT". $)
```
Comment (tirix) [inline iset.mm:16]: You miss a "T": /SPLI**T**

File: iset.mm [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
-   understand the label naming conventions used in iset.mm.  See also "help
+$( New users may want to read https://us.metamath.org/ileuni/conventions.html
+   to understand the label naming conventions used in iset.mm.  See also "help
    verify markup" in the Metamath program for markup conventions. $)
 
-$( To break this file into smaller modules, in the Metamath program type "read
-   iset.mm" followed by "write source iset.mm /split"; to recombine, omit
-   "/split". $)
+$( To break this file into smaller modules, in the Metamath program type
+   "MM> READ iset.mm" followed by "MM> WRITE SOURCE iset.mm / SPLI".  To
```
Comment (benjub) [inline iset.mm:15]:
```suggestion
   "MM> READ iset.mm" followed by "MM> WRITE SOURCE iset.mm / SPLIT".  To
```

Comment (benjub) [reply iset.mm:15]: I cannot apply it. Maybe you can ?

Comment (benjub) [reply iset.mm:15]: (I keep seeing "This diff has recently been updated. Please refresh and try again." but refreshing has no effect but returning to the initial stage; this seems to be a bug in github.)

Comment (tirix) [reply iset.mm:15]: I get the same message,

Comment (tirix) [reply iset.mm:15]: I did a "manual" PR, could not find any other way.

**Review by tirix — APPROVED** [review]

File: iset.mm (hunk 2 above)
Comment (benjub) [reply iset.mm:15]: Thanks, and sorry for the inconvenience !

**Pull request merged**